engine/websocketroutine_manager.go

```diff
@@ -299,29 +288,6 @@ func (m *websocketRoutineManager) FormatCurrency(p currency.Pair) currency.Pair
 		m.currencyConfig.CurrencyPairFormat.Uppercase)
 }
 
-// printOrderChangeSummary this function will be deprecated when a order manager
-// update is done.
-func (m *websocketRoutineManager) printOrderChangeSummary(o *order.Modify) {
```

Comment:
Please keep this and update it. You've changed all WS modifies to Detail and every message says the order is new. Update handling to print this when modifying an existing order and use the other one for printing a new order